src/EventTickets/resources/admin/components/EventDetailsPage/InnerPageListTable/index.tsx

```diff
@@ -0,0 +1,125 @@
+import {__} from '@wordpress/i18n';
+import cx from 'classnames';
+import {ListTable} from '@givewp/components';
+import A11yDialogInstance from 'a11y-dialog';
+import ListTableStyles from '@givewp/components/ListTable/ListTablePage/ListTablePage.module.scss';
+import {createContext, useRef, useState} from 'react';
+import ListTableApi from '@givewp/components/ListTable/api';
+import {useResetPage} from '@givewp/components/ListTable/hooks/useResetPage';
+import {A11yDialog} from 'react-a11y-dialog';
+import styles from './InnerPageListTable.module.scss';
+
+export const ShowConfirmModalContext = createContext((label, confirm, action, type = null) => {});
+
+export default function InnerPageListTable({apiSettings, listTableBlankSlate, rowActions, singleName, pluralName, title}) {
+    const [page, setPage] = useState<number>(1);
+    const [selectedIds, setSelectedIds] = useState([]);
+    const [selectedNames, setSelectedNames] = useState([]);
+    const [modalContent, setModalContent] = useState<{
+        confirm;
+        action;
+        label;
+        type?: 'normal' | 'warning' | 'danger';
+    }>({
+        confirm: (selected) => {},
+        action: (selected) => {},
+        label: '',
+    });
+    const dialog = useRef() as {current: A11yDialogInstance};
+    const [sortField, setSortField] = useState<{sortColumn: string; sortDirection: string}>({
+        sortColumn: 'id',
+        sortDirection: 'asc',
+    });
+    const {sortColumn, sortDirection} = sortField;
+
+    const parameters = {
+        page,
+        perPage: 30,
+        sortColumn,
+        sortDirection,
+        locale: navigator.language,
+        testMode: undefined,
+    };
+
+    const archiveApi = useRef(new ListTableApi(apiSettings)).current;
+
+    const {data, error, isValidating, mutate} = archiveApi.useListTable(parameters);
+
+    useResetPage(data, page, setPage, {});
+
+    const showConfirmActionModal = (label, confirm, action, type: 'normal' | 'warning' | 'danger' | null = null) => {
+        setModalContent({confirm, action, label, type});
+        dialog.current.show();
+    };
+
+    const setSortDirectionForColumn = (column, direction) => {
+        setSortField((previousState) => {
+            return {
+                ...previousState,
+                sortColumn: column,
+                sortDirection: direction,
+            };
+        });
+    };
+
+    return (
+        <>
+            <ShowConfirmModalContext.Provider value={showConfirmActionModal}>
+                <div className={styles.innerPageListTable}>
+                <ListTable
```

Comment:
Do we need the full list table component? Ticket Types will most likely be a single page, so no pagination, probably doesn't need column sorting or column filtering by search.

Can we just localize a list of ticket types to per-populate a table? Adding a new event can re-load the page.

Reply:
Refactored. Also removed some components that had been introduced here but not used and that will be heavily changed in next PRs like: `InnerPageListTable`, `EventSectionRowActions` and `TicketTypesRowActions`.